jerry.sh

```diff
@@ -1038,6 +1041,29 @@ read_manga() {
 
 }
 
+immediate_score() {
+    i_total="0"
+    if [ $1 = "anime" ]; then
+	    i_total="$episodes_total"
+    elif [ $1 = "manga" ]; then 
+	    i_total="$chapters_total"
+    else
+	    return
+    fi
+
+    if [ $(($progress + 1)) = $(echo $i_total | cut -d " " -f1) ]; then
+	printf "Enter a score: "
+	read -r i_score
+    else
+	return
+    fi
```

Comment:
Why would you need to ask for episode/chapter number? It will always be the last episode/chapter since the score is only requested when you finish the whole anime or manga.

Reply:
yes, my mistake, please ignore the part where i said that, not sure what i was thinking